tests/python/test_deepseek_v3.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025-present NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import transformers
+import torch
+from contextlib import contextmanager
+
+
+@contextmanager
+def default_tensor_type(dtype=torch.float32, device="cpu"):
+    # Save
+    prev_dtype = torch.get_default_dtype()
+    prev_device = torch.get_default_device()
+
+    # Set
+    torch.set_default_dtype(dtype)
+    torch.set_default_device(device)
+
+    yield
+
+    # Restore
+    torch.set_default_dtype(prev_dtype)
+    torch.set_default_device(prev_device)
+
+
+def test_transformer_layer():
+    config = transformers.AutoConfig.from_pretrained(
```

Comment:
Have we put in a requirement anywhere to reflect that our tests require the "transformers" package?

Reply:
Good question. Since it's only used in tests not by nvFuser, I prefer not putting it in a requirement. If the user doesn't have `transformers` installed, they'll see a unit test failure at most -- nvFuser itself will still be functional. 

PS: https://github.com/NVIDIA/Fuser/blob/main/requirements.txt doesn't even have torch. Isn't that a problem?